lib/local_daily_digest.js

```diff
@@ -30,18 +39,61 @@ function get(masterCb) {
 				lastDigestSent = new Date(lastDigestSent);
 				var offset = current.valueOf() - lastDigestSent.valueOf();
 				if (offset > 24 * 60 * 60 * 1000) { // If last digest was sent more than 24 hours ago
+					//if profile roles does not contain manager or editor
 	  				allLocalProfilesWDigest.push(profile);
+	  			
+	  				if(!(matchInArray('manager:', profile.roles)) && !(matchInArray('editor:', profile.roles)))
```

Comment:
I might be wrong, but I believe the logic is wrong here: take the case of someone who is a manager of location X, but also subscribed to the digest for location Y. What will he receive ? I believe that currently, he will receive the normal digest (ie not the org one) for location X AND location Y...